oracle/oracle.go

```diff
@@ -444,3 +458,125 @@ func (o *oracleOps) Delete(volumeID string) error {
 	}
 	return nil
 }
+
+func (o *oracleOps) SetInstanceGroupSize(instanceGroupID string, count int64, timeout time.Duration) error {
+
+	ctx := context.Background()
+	var cancel context.CancelFunc
+	if timeout > time.Nanosecond {
+		// If timeout is non-zero then
+		// add deadline to the operation else
+		// wait infinitely for operation to complete
+		ctx, cancel = context.WithTimeout(ctx, timeout)
```

Comment:
this `ctx` object is not used anywhere (in ListNodePools or in UpdateNodePool request)

Reply:
Removed